src/MobileUI/Pages/LeaderboardPage.xaml

```diff
@@ -20,38 +20,37 @@
         <mct:StatusBarBehavior StatusBarColor="{StaticResource Background}"
           StatusBarStyle="LightContent" />
     </ContentPage.Behaviors>
-    <Grid>
+    <Grid RowDefinitions="Auto,*">
+        <controls:SegmentedControl
+            Grid.Row="0"
+            Segments="{Binding Periods}"
+            HeightRequest="50"
+            Padding="15,10,15,0"
```

Comment:
1. Can you please also add bottom padding to the `SegmentedControl` so that there is some space around it similar to what we have on the NetworkPage.

2. When the page loads, `SegmentedControl` is displayed as an empty box. Can we please initialise `Periods` before we make the API call.

Reply:
All done - the SegmentedControl on the Leaderboard page should now match that of the Network page, and the Periods load first so they don't show blank on initial load.

![Screenshot 2024-03-28 at 12 09 05 PM](https://github.com/SSWConsulting/SSW.Rewards.Mobile/assets/11418832/9d63cff9-cee1-4582-809d-e3474b9ea482)

**Figure: Updated SegmentedControl**

Reply:
Fixed the padding now.

![Screenshot 2024-03-28 at 12 31 18 PM](https://github.com/SSWConsulting/SSW.Rewards.Mobile/assets/11418832/9184fcc5-2732-4198-b6fc-b5f17967515f)

**Figure: Adjusted padding**